pkgs/applications/science/biology/bedops/default.nix

```diff
@@ -18,7 +18,19 @@ stdenv.mkDerivation rec {
     # We use nixpkgs versions of these libraries
     rm -r third-party
     sed -i '/^LIBS/d' system.mk/*
-    sed -i 's|^LIBRARIES.*$|LIBRARIES = -lbz2 -lz -ljansson|' */*/*/*/Makefile
+    sed -i 's|^LIBRARIES.*$|LIBRARIES = -lbz2 -lz -ljansson|' */*/*/*/Makefile*
+
+    # `make support` installs above libraries
+    sed -i 's/: support/: /' system.mk/*
+
+    # Variable name is different in this makefile
+    substituteInPlace applications/bed/sort-bed/src/Makefile.darwin \
+      --replace "DIST_DIR" "BINDIR"
+
+    # `mkdir -p $BINDIR` is missing
+    substituteInPlace applications/bed/sort-bed/src/Makefile.darwin \
+      --replace 'mkdir -p ''${OBJ_DIR}' 'mkdir -p ''${OBJ_DIR} ''${BINDIR}'
+
```

Comment:
I think simply doing that in `preInstall` (with a short comment why) would be better.

Reply:
I think that `preBuild` is the correct phase to use, as this process is necessary before to the actual build.

Reply:
I meant to simply run `mkdir` in `preInstall` rather than patching the build.

Reply:
I need this `BINDIR` at build time, not at install time. Otherwise, `sort-bed` fails to build.